config/locales/en.yml

```diff
@@ -1200,7 +1225,7 @@ en:
       increased page size. This number doesn't necessarily reflect the
       amount of new content the user added. Restoring removed content
       will also increase this total.
-    chars_added: Characters Added<br>(article | user | draft)
+    chars_added: Characters Added<br>(item | user | draft)
```

Comment:
It looks like this changes the default version rather than add a new variation for Wikidata.

Reply:
I thought that  ' characters added' is only for Wikidata, but it is for all non-Wikipedias, right? I'll change that.

Reply:
Characters added can be for any project. For English Wikipedia specifically, we use 'words added' instead in some places (by converting a byte/character count to an estimated word count), but the data comes from basically just the difference in byte size between two revisions of a wiki page.